cookbooks/vm/recipes/git.rb

```diff
@@ -20,8 +20,3 @@
   mode '0644'
   action :create_if_missing
 end
-
-bashd_entry 'setup-git-ps1-prompt' do
```

Comment:
Unfortunately not, the bashrc_manager resource doesn't offer to parse and store template files. One could parse the template manually into a string and pass that to the bashrc_manager resource.
My ruby skills are not good enough to implement that myself, maybe you can help me with that?

Reply:
Found a workaround :-)

Reply:
heh, nice! :)

Reply:
On second look: I have no clue why I added the `git_ps1.erb` as a template, because there are no interpolated variables in it at all. Oh yes I know, it was because the former [bashd cookbook](https://github.com/jakshi/bashd) was an .erb file...

Since it is not templated anyway, you could as well
a) define the whole content inline (I would probably use a [squiggly heredoc](http://www.virtuouscode.com/2016/01/06/about-the-ruby-squiggly-heredoc-syntax/))
b) simply use `IO.read(filename)` instead of the newly added `from_template` function

Reply:
I went for b) and updated the pull request